### DISCUSSION TITLE
Ssh into Termux?

### DISCUSSION THREAD

**Discussion by julkhami** [termux/termux-app#2333] [Q&A]
Hey,

I’d really like some help about ssh’ing into Termux.

I followed the steps on the Wiki but my big problem is that when I start sshd and see the output in logcat it doesn’t say that the server is listening on a port.

Does anyone know what we have to do in our network settings to get the sshd server running apart from just calling “ssh”?

**Comment by Bezborodoff** [comment]
Может это поможет: sshd -e -d -d -d

**Comment by julkhami** [comment]
Большое спасибо, это работает. Есть ли способ предотвратить отключение
ssh-сервера и его перезапуск?
